src/app/pages/vm/devices/device-form/device-form.component.spec.ts

```diff
@@ -658,8 +659,31 @@ describe('DeviceFormComponent', () => {
           Port: '5900',
           Resolution: '1024x768',
           'Web Interface': true,
+          'Web Port': '5901',
         });
       });
+
+      it('shows web_port field only when Web Interface is enabled', async () => {
+        // Initially web is true, so web_port should be visible
+        let values = await form.getValues();
+        expect(values).toHaveProperty('Web Port', '5901');
```

Comment:
@william-gr we have `vm.port_wizard` endpoint which gives default values which are valid and won't raise a validation error. So we should use this on creation of a display device

Reply:
In the UI I am expecting people to leave it empty to auto assign a port